package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "react-global-store",
+  "version": "1.0.0",
+  "description": "A simple global state solution for React",
+  "keywords": [
+    "react",
+    "store"
+  ],
+  "main": "dist/main.js",
+  "scripts": {
+    "build": "webpack --mode production",
+    "lint": "eslint ./src ./test",
+    "test": "yarn build && yarn lint && yarn test:unit",
+    "test:unit": "nyc mocha",
+    "watch": "yarn storybook",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "author": "Jonathan Haines",
+  "license": "MIT",
+  "dependencies": {
+    "lodash": "^4.17.10",
```

Comment:
lodash kind of seems unnecessary for such a small lib imo - but if its too much rework never mind